.drone/drone.yml

```diff
@@ -399,6 +399,185 @@ steps:
 depends_on:
 - check
 
+---
+kind: pipeline
+name: docker-ppc64le
+
+platform:
+  os: linux
+  arch: ppc64le
```

Comment:
@carlosedp this has broke our CI? What do you recommend ? I don't think we want to run drone off a fork. Why did you need this ? Was it mainly to get a makefile working with this arch in this case may be we can revert the pipeline ??

Not sure but we can't stay with broken CI for a long time.